companies/management/commands/sync_corp_codes.py

```
@@ -122,6 +128,302 @@ def add_arguments(self, parser):
+        # STEP 2: 종목-업종 매핑을 메모리 딕셔너리로 생성 (테이블 사용 안 함)
+        self.stdout.write("STEP 2: 종목-업종 매핑 메모리 적재 중...")
+        # ticker -> kis_code 매핑 딕셔너리
+        ticker_to_kis = {}
+
+        if not dry_run:
+            mapping_count = 0
+            for target in [
+                {"name": "KOSPI", "file": "kospi_code.mst.zip"},
+                {"name": "KOSDAQ", "file": "kosdaq_code.mst.zip"},
+            ]:
+                res_stk = requests.get(base_url + target["file"])
```

Ruff (0.14.13)

Probable use of `requests` call without timeout

(S113)